ts/features/bonus/cgn/__e2e__/cgn.e2e.ts

```diff
@@ -39,13 +60,17 @@ describe("CGN", () => {
     });
   });
 
+  /*
+  TODO: we should deactivate atm this test because we cannot scroll in the WalletHome, since is used native-base and we cannot have a testID binded to the scrollview.
+
   describe("When the user want to start activation from card carousel", () => {
     it("Should complete activation", async () => {
       await element(by.text(I18n.t("global.navigator.wallet"))).tap();
       await element(by.id("FeaturedCardCGNTestID")).tap();
       await activateBonusSuccess();
     });
   });
+   */
```

Comment:
I've tried to add the `testID` at https://github.com/pagopa/io-app/blob/361054dbd30325e1f1195eed68779ea633556653/ts/screens/wallet/WalletHomeScreen.tsx#L365 but it seems too fail. I get the error:

```
    DetoxRuntimeError: Test Failed: View “<RCTView: 0x7f8b91704870>” is not visible: View does not pass visibility percent threshold (100%)

    HINT: To print view hierarchy on failed actions/matches, use log-level verbose or higher.

      64 |     it("Should complete activation", async () => {
      65 |       await element(by.text(I18n.t("global.navigator.wallet"))).tap();
    > 66 |       await element(by.id(`WalletCardsTestID`)).swipe("up");
         |                                                 ^
      67 |       await element(by.id("FeaturedCardCGNTestID")).tap();
      68 |       await activateBonusSuccess();
      69 |     });

      at _callee2$ (ts/features/bonus/cgn/__e2e__/cgn.e2e.ts:66:49)
      at tryCatch (node_modules/regenerator-runtime/runtime.js:63:40)
```
Maybe I'm missing some step?

Reply:
Mmmm not actually....

Try to add this instruction before the swipe:
```ts
await waitFor(element(by.id("WalletCardsTestID")))
    .toBeVisible()
    .withTimeout(e2eWaitRenderTimeout);
```

It could be a timing issue.

Reply:
I've also tried to wait but I always receive the same error.
In https://github.com/pagopa/io-app/pull/3944/commits/0c727703b6a87ccca72dda5efb079925874354fd I've used the same idea, but with `by.text(I18n.t("wallet.paymentMethods")` and this seems to work. 
It should be noted that this is a temporary workaround that could stop working if we will add more e2e tests that adds more payment methods, until we have a way to reset the dev-server state.